src/core/libmaven/PeakDetector.cpp

```diff
@@ -463,6 +463,14 @@ void PeakDetector::identifyFeatures(const vector<Compound*>& identificationSet)
                     continue;
                 }
 
+                // we should filter the annotated group based on its RT, if the
```

Comment:
`filterByMS2` is computationally heavier than the RT check. Maybe switch them around? just a suggestion.

Reply:
Good suggestion. Changed locally.